#ifndef INVERSEKINEMATICSNODE_H
#define INVERSEKINEMATICSNODE_H

#include "rclcpp/rclcpp.hpp"

using std::placeholders::_1;
class InverseKinematicsNode: public rclcpp::Node
{
private:
    //Private methods
    //Private attributes
public:
    //Public methods
    InverseKinematicsNode();
};


#endif